### ISSUE TITLE
0.10.0 beaks api w/ 0.8.0

### ISSUE THREAD

**Issue by senior7515** [hakuch/CMakeCooking#16]
invoking cooking.sh -r foo no longer works. only cooking.sh -r <PATH>/foo  works which breaks the previous API so now we have to modify all scripts that depended on this. 

Wonder if we should set up some CI for this so we don't break downstream projects w/ minor revisions.

**Comment by senior7515** [comment]
i.e.: 

```
Cooking: The '/home/agallego/workspace/smf/wellknown' recipe does not exist!

```

**Comment by hakuch** [comment]
cmake-cooking is pre 1.0, so API breakages are expected and documented. Perhaps they can be documented more clearly, but that's a separate issue. :-)

This particular issue is described in the `CHANGES.md` for the 0.10.0 release.

**Comment by hakuch** [comment]
Looking at `CHANGES.md`, it's evident that it's not clear enough that the old behavior changed. I'll try to keep this in mind in the future.

**Comment by hakuch** [comment]
I think you're right that this change changed the behavior too much.

I'll restore the old behavior (of resolving to `recipe/${name}.cmake` and we can reconsider this change for a future release.

**Comment by hakuch** [comment]
Fixed via 206a31d9625313fbc4f36a08fd23708e413d7aa7.
